Assets/MRTK/Providers/WindowsMixedReality/XR2018/Controllers/BaseWindowsMixedRealitySource.cs

```diff
@@ -96,6 +95,12 @@ public virtual void UpdateController(InteractionSourceState interactionSourceSta
             }
         }
 
+        public void UpdateSixDofData(InteractionSourceState interactionSourceState)
```

Comment:
It looks like this is only called by derived classes, and the equivalent in the XRSDK controller is `protected`:

```suggestion
        protected void UpdateSixDofData(InteractionSourceState interactionSourceState)
```